frontend/src/metabase/setup/components/Setup.jsx

```diff
@@ -202,6 +203,12 @@ export default class Setup extends Component {
             </div>
           </div>
 
+          <DriverWarning
+            engine={selectedDatabaseEngine}
+            ml={26}
+            style={{ marginBottom: "8px" }}
+            data-testid={"database-setup-driver-warning"}
+          />
```

Comment:
The `DriverWarning` is invisible now because of the page layout. You can move it under the `AddDatabaseHelpCardHolder` and remove the left margin

Reply:
Ah yeah, I accidentally left the raw style here after fixing it in the CSS.  Fixed in the next push.